ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/DescribeConnectorExecutor.java

```diff
@@ -75,22 +78,26 @@ public StatementExecutorResponse execute(
         .getConnectClient()
         .status(connectorName);
     if (statusResponse.error().isPresent()) {
-      return StatementExecutorResponse.handled(Optional.of(
-          new ErrorEntity(
-              configuredStatement.getStatementText(),
-              statusResponse.error().get())
-      ));
+      throw new KsqlRestException(EndpointResponse.create()
+          .status(statusResponse.httpCode())
+          .entity(new KsqlErrorMessage(
+              Errors.toErrorCode(statusResponse.httpCode()),
+              "Failed to query connector status: " + statusResponse.error().get()))
```

Comment:
nit: move errorMsg to a final var for consistency and readability -- applies to all KsqlRestExceptions

Reply:
Hm interesting. I think most places in the codebase inline the error message directly rather than having a separate line for `final String errorMsg = ...;` but I do see a few places where we opt for the latter and are inconsistent as a result. For relatively short error messages such as this one, I don't personally think it's harder to read when the error message is inlined but I don't feel too strongly. I'll make the change.